index.js

```diff
@@ -31,6 +31,14 @@ function S3StreamLogger(options){
     this.acl                    = options.acl || false;
     this.compress               = options.compress || false;
 
+    // Validating input storage_class parameter & setting default storage_class - STANDARD
+    options.storage_class = options.storage_class || 'STANDARD';
+    if(['STANDARD', 'REDUCED_REDUNDANCY', 'STANDARD_IA', 'ONEZONE_IA','INTELLIGENT_TIERING', 'GLACIER', 'DEEP_ARCHIVE'].indexOf(options.storage_class) > -1){
```

Comment:
I think it's better not to validate the parameter here - the value will be validated by the S3 library anyway,

Reply:
actually also - if the option value is not set, then we should also not set it on the S3 call (in case for some reason the default value used aws-sdk changes), i.e. the same way the other options work at the moment

Reply:
So I did have a round of dev testing and one particular scenario throws an error. 

Supposing the storage_class key is set to either of [true, {name: project},-2] it throws the following error
`message: 'Expected params.StorageClass to be a string',
  code: 'InvalidParameterType',
`

So i guess its better to have some sort of validation like
`if(typeof options.storage_class === 'string')`

Do suggest.